examples/llm/inference/benchmarking/mcloud/benchmark_launcher.yaml

```diff
@@ -0,0 +1,24 @@
+run_name: deepspeed-inference
+gpu_num: 8
+cluster: r0z0 # Update with your cluster here!
+image: mosaicml/composer:0.13.2
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  pip_install: '.[llm]'
+
+- integration_type: git_repo
+  git_repo: mosaicml/composer
+  # Right now composer 0.13.2 might not have the right HF changes right now
+  # So this is the current commit hash
+  git_commit: 0c704d31ead5963f57b3d5361a19666937a7e2d9
+  pip_install: '.[all]'
+
+command: |
+  cd examples/examples/llm/inference/benchmarking/
+
+  # For now since we are in the single GPU setting
```

Comment:
Can we just drop this comment and make sure we note somewhere (Jira) whatever the issue is? The comment right now is pretty confusing for a user. A new comment can just say this is single GPU for now.